src/components/10-atoms/button/index.js

```diff
@@ -44,7 +44,7 @@ class AXAButton extends LitElement {
       'a-button--large': this.large,
       'a-button--secondary': this.secondary,
       'a-button--inverted': this.inverted,
-      'a-button--motion': this.motion,
+      'a-button--motion': !this.static,
```

Comment:
I would suggest to use still or similar, not a word that has a meaning in Javascript (like static has). Users cant get confused. If is a NON JS word, then maybe is clearer what we want to achive. is not a show stopper thought so ur desicion :)